### ISSUE TITLE
Minimum version is now 1.49.0?

### ISSUE THREAD

**Issue by chipsenkbeil** [async-graphql/async-graphql#468]
## Expected Behavior

Compiling on Rust 1.46.0 succeeds.

## Actual Behavior

Compilation fails about some unstable feature regarding some library getting pulled in using [spinloop](https://doc.rust-lang.org/std/hint/fn.spin_loop.html), which did not stabilize until 1.49.0. This seems to be due to a newer version of a library being pulled in as part of async_graphql, even on older versions such as 2.4.6.

```
error[E0658]: use of unstable library feature 'renamed_spin_loop'
  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/spin-0.9.0/src/relax.rs:26:9
   |
26 |         core::hint::spin_loop();
   |         ^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #55002 <https://github.com/rust-lang/rust/issues/55002> for more information

error: aborting due to previous error
```

## Steps to Reproduce the Problem

1. Install Rust 1.46.0 via `rustup toolchain install --minimal 1.46.0`
2. Create some project that uses async_graphql (version 2.4.6 or greater? not sure how far back it goes)
3. Attempt to compile via `cargo +1.46.0 build`

## Specifications

- Version: 2.7.4 (although this started happening for me with older versions all the way back to 2.4.6)
- Platform: Windows/Linux/Mac
- Subsystem: ???

**Comment by chipsenkbeil** [comment]
I didn't see automation covering the minimum Rust version highlighted in your README. If you aren't doing it yet, I'd recommend tossing that into your CI as I've seen it done for a variety of Rust crates as a best practice.

Here's one of mine for example: https://github.com/chipsenkbeil/entity-rs/blob/main/.github/workflows/ci.yml

It's how I noticed that my project started failing for Rust 1.48.0, which was my minimum version until I bumped to 1.49.0 to fix the dependency breakage. See [past failures](https://github.com/chipsenkbeil/entity-rs/actions) in my CI for examples of failing due to the spinloop. `cargo tree` highlighted to me that the dependency was coming from async_graphql.
